packages/typography/index.demo.tsx

```diff
@@ -0,0 +1,36 @@
+import { h, Component } from 'skatejs';
```

Comment:
I have tried that at first, but it was huge file, which was not very readable, so I restored the demo files.